src/ethereum/EthereumTools.ts

```diff
@@ -357,8 +365,35 @@ export class EthereumTools implements EdgeCurrencyTools {
     }
     return cleanLocation.contractAddress.toLowerCase().replace(/^0x/, '')
   }
+
+  // #region otherMethods
+
+  /**
+   * Resolve an ENS name, for example: "bob.eth"
+   */
+  async resolveEnsName(ensName: string): Promise<string> {
```

Comment:
Why not make this called `resolveName(name: string): Promise<string>` just incase other name services are deployed besides ENS?

Reply:
This implementation is specific to ENS since it only works for ENS on ETH. If we deploy other name services that can be generic (including support in ETH), then we can do this

Reply:
Fair enough, though I'd imagine the function signature would be the same. I only thought that maybe if we make it generic and just fail on the input if it's not an ENS name (or whatever the implementation supports), then there wouldn't be a need for a new method later on. It's forward thinking, so not necessary.